src/objects/zcl_abapgit_objects_super.clas.abap

```diff
@@ -306,6 +308,14 @@ CLASS zcl_abapgit_objects_super IMPLEMENTATION.
 
         CONCATENATE 'adt://' sy-sysid <lv_uri> INTO lv_adt_link.
 
+        IF i_line_number IS NOT INITIAL.
+          lv_line_number = i_line_number.
+          SHIFT lv_line_number LEFT DELETING LEADING ' 0'.
+          CONDENSE lv_line_number NO-GAPS.
+          CONCATENATE lv_adt_link '/source/main#start=' lv_line_number ',0;end=' lv_line_number ',1 ' INTO lv_adt_link.
```

Comment:
did you test with both CLAS and PROG? as far as I remember the links are different for the 2 object types

Reply:
Works fine now for all objects